packages/core/src/components/tabs/tabs.tsx

```diff
@@ -316,5 +316,5 @@ function isEventKeyCode(e: React.KeyboardEvent<HTMLElement>, ...codes: number[])
 }
 
 function isTab(child: React.ReactChild): child is TabElement {
```

Comment:
when finished, should be able to delete this function and use `Utils.isElementType(child, Tab)` directly.